include/sound/sof/dai-intel.h

```diff
@@ -49,6 +49,9 @@
 /* bclk idle */
 #define SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_IDLE_HIGH	BIT(5)
 
+/* DMIC max. four controllers for eight microphone channels */
+#define SOF_DAI_INTEL_DMIC_NUM_PDM			4
```

Comment:
NUM_DMIC_CTRL? that would avoid ambiguities with channels?

Reply:
That sounds like a good proposal!

Reply:
Done